src/assets/styles/base/_layout.scss

```diff
@@ -31,8 +31,16 @@ body {
 }
 
 .container {
-  width: 90%;
+  width: 100%;
```

Comment:
Oké, we're in the early stages of design, but shall we think about mobile first? A 100% container will mean it stretches the full width, until it reaches max width. That is not the purpose of the container; it is to keep stuff "off" the edges and center the content.

Reply:
I will use the .container class to handle max-width and centering. Another class will handle paddings